.github/workflows/e2e-ci.yaml

```diff
@@ -256,6 +256,7 @@ jobs:
           APPINSIGHTS_INSTRUMENTATIONKEY: ${{ secrets.APPINSIGHTS_INSTRUMENTATIONKEY }}
           CERT_REMOTE_PATH: ${{ needs.env_var.outputs.CertRemotePath }}
           SERVER_PFX_PASSWORD: ${{ secrets.SERVER_PFX_PASSWORD }}
+          NET_SRV_LOG_TO_TCP_ADDRESS: "itestup"
```

Comment:
Is this really needed?

Reply:
If we don't set this here, the template won't be able to replace the value, therefore its execution on AIO will fail